### PR TITLE
Female npc about issue #338

### PR DESCRIPTION
**Pull request by omattornot**
Hy !
Turn base npc into female. One commit by npc.

### CODE REVIEW

**Comment by omattornot** [comment]
Ok, i think this is enough for me, principals woman npc for me ^^

**Comment by mekolat** [comment]
noooooooooo not my Rouge :cry:

**Comment by omattornot** [comment]
You prefer the short ? :( (really, if Rouge has a slip, that match with stocking, but the short...)

**Comment by wushin** [comment]
@omattornot @mekolat wants rouge left alone. No changes needed to that NPC.

**Comment by omattornot** [comment]
Ok :) is back to base model.

**Comment by wushin** [comment]
@omattornot it would be better to rebase and remove both the npc181 commits.

**Comment by omattornot** [comment]
It's my first rebase, i hope this is good.

**Comment by wushin** [comment]
@omattornot looks good

File: graphics/sprites/equipment/legs/skirt-female.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
 <sprite variants="135" variant_offset="1">
-  <imageset name="base" src="graphics/sprites/equipment/legs/skirt-male.png|W" width="64" height="64"/>
+  <imageset name="base" src="graphics/sprites/equipment/legs/skirt.png|W" width="64" height="64"/>
```
Comment (wushin) [inline graphics/sprites/equipment/legs/skirt-female.xml]: @omattornot revert this change out as we need art as well #341 I've fixed the skirt here.

**Comment by mekolat** [comment]
the commit messages are misleading, as you're not adding any new npcs, you're changing their gender. you might want to re-write your git history (`git rebase -i HEAD~9` then change `pick` for `r`)

**Comment by omattornot** [comment]
Ahah, yes your right. I change this.
